src/faq/plugin.ts

```diff
@@ -33,98 +33,286 @@ const faqExtension = {
+
+  basicsQA.appendChild(basicsQ4);
+  basicsQA.appendChild(basicsQ4Ans);
+
+  let tourButton = document.createElement('a');
+  tourButton.className = 'faqa';
+  let tourButtonText = document.createTextNode('here');
+  tourButton.appendChild(tourButtonText);
+  tourButton.addEventListener('click', () => {
+    app.commands.execute('about-jupyterlab:show');
+  });
+
+  let basicsQ4AnsLeft = document.createTextNode('!');
+
+  basicsQ4Ans.appendChild(tourButton);
+  basicsQ4Ans.appendChild(basicsQ4AnsLeft);
+
+
+  let featuresHeading = document.createElement('h2');
+  featuresHeading.className = 'faqh2';
+  featuresHeading.textContent = 'FEATURES';
+  faqContent.appendChild(featuresHeading);
+
+  let featuresQA = document.createElement('ul');
+  featuresQA.className = 'faqul';
+
+  let featuresQ1 = document.createElement('li');
+  featuresQ1.className = 'faqquestion';
+  featuresQ1.textContent = 'How do I add more kernels/languages to JupyterLab?';
+  let featuresQ1Ans = document.createElement('li');
+  featuresQ1Ans.className = 'faqanswer';
+  featuresQ1Ans.textContent = 'To add more languages to the JupyterLab you must install '
+   + 'a new kernel. Installing a kernel is usually fairly simple and can be done '
+   + 'with a couple terminal commands. However the instructions for installing kernels '
+   + 'is different for each language. For further instructions, click ';
+
+  let kernelLink = document.createElement('a');
+  kernelLink.className = 'faqa';
+  let kernelLinkText = document.createTextNode('this');
+  kernelLink.appendChild(kernelLinkText);
+  kernelLink.href = 'http://jupyter.readthedocs.io/en/latest/install-kernel.html';
+  kernelLink.target = '_blank';
+  featuresQ1Ans.appendChild(kernelLink);
+
+  let featuresQ1AnsLeft = document.createTextNode(' link');
+  featuresQ1Ans.appendChild(featuresQ1AnsLeft);
+
+  faqContent.appendChild(featuresQA);
+  featuresQA.appendChild(featuresQ1);
+  featuresQA.appendChild(featuresQ1Ans);
+
+  let featuresQ2 = document.createElement('li');
+  featuresQ2.className = 'faqquestion';
+  featuresQ2.textContent = 'How can I share my notebooks?';
+  let featuresQ2Ans = document.createElement('li');
+  featuresQ2Ans.className = 'faqanswer';
+  featuresQ2Ans.textContent = 'You can either publish your notebooks on GitHub or use a '
+   + 'free service such as ';
+
+  let nbViewerLink = document.createElement('a');
+  nbViewerLink.className = 'faqa';
+  let nbViewerLinkText = document.createTextNode('nbviewer.org');
+  nbViewerLink.appendChild(nbViewerLinkText);
+  nbViewerLink.href = 'https://nbviewer.jupyter.org/';
+  nbViewerLink.target = '_blank';
+  featuresQ2Ans.appendChild(nbViewerLink);
+
+  let featuresQ2AnsLeft = document.createTextNode(' to render your notebooks online.');
+  featuresQ2Ans.appendChild(featuresQ2AnsLeft);
+
+  featuresQA.appendChild(featuresQ2);
+  featuresQA.appendChild(featuresQ2Ans);
+
+
+  let developerHeading = document.createElement('h2');
```

Comment:
I'd say use comments to break up the seconds rather than just extra whitespace (e.g. `// Create developer section.`)